catalog/resource_catalog.go

```diff
@@ -27,15 +27,16 @@ func ucDirectoryPathSlashAndEmptySuppressDiff(k, old, new string, d *schema.Reso
 }
 
 type CatalogInfo struct {
-	Name          string            `json:"name"`
-	Comment       string            `json:"comment,omitempty"`
-	StorageRoot   string            `json:"storage_root,omitempty" tf:"force_new"`
-	ProviderName  string            `json:"provider_name,omitempty" tf:"force_new,conflicts:storage_root"`
-	ShareName     string            `json:"share_name,omitempty" tf:"force_new,conflicts:storage_root"`
-	Properties    map[string]string `json:"properties,omitempty"`
-	Owner         string            `json:"owner,omitempty" tf:"computed"`
-	IsolationMode string            `json:"isolation_mode,omitempty" tf:"computed"`
-	MetastoreID   string            `json:"metastore_id,omitempty" tf:"computed"`
+	Name           string            `json:"name"`
+	Comment        string            `json:"comment,omitempty"`
+	StorageRoot    string            `json:"storage_root,omitempty" tf:"force_new"`
+	ProviderName   string            `json:"provider_name,omitempty" tf:"force_new,conflicts:storage_root"`
+	ShareName      string            `json:"share_name,omitempty" tf:"force_new,conflicts:storage_root"`
+	ConnectionName string            `json:"connection_name,omitempty" tf:"force_new,conflicts:storage_root"`
+	Properties     map[string]string `json:"properties,omitempty"`
```

Comment:
CatalogInfo also has options - our open api spec needs updating.

Reply:
cc @Doug-Luce

Reply:
@nkvuong Can you address this one as well? Thanks!

Reply:
@andrewli81 how would the options be split between connections & catalogs? and would there be sensitive options to be handled here as well? 
once this is fixed in OpenAPI, we just need to regenerate the SDK, so should not take long

Reply:
@nkvuong 

> @andrewli81 how would the options be split between connections & catalogs? and would there be sensitive options to be handled here as well?

The options of catalogs are mainly used to identify databases. This is because database systems like postgresql uses 3 layer namespaces (same as UC), so customer will need to specify which database to import into a UC catalog. 

https://docs.databricks.com/en/query-federation/postgresql.html#language-SQL:~:text=Run%20the%20following%20SQL%20command%20in%20a%20notebook%20or%20Databricks%20SQL%20editor.%20Items%20in%20brackets%20are%20optional.%20Replace%20the%20placeholder%20values%3A

The open API spec has been fixed: https://docs.databricks.com/api/workspace/connections/create

Can we also fix it here?